configure

```diff
@@ -285,10 +285,10 @@ elif (args.comp == "clang"): f.write("COMP = CLANG\n")
 elif (args.comp == "intel"): f.write("COMP = INTEL\n")
 message("Comp-Type", str(args.comp))
 shellmessage("Comp-Version",args.comp+" --version")
-shellmessage("Comp-MPI","mpicxx -cxx="+args.comp+" -show")
-f.write("CC = mpicxx -cxx="+args.comp+"\n")
+shellmessage("Comp-MPI","mpicxx -show")
```

Comment:
What is the reason for this change? Users may desire to use another compiler (intel or clang) and this would remove that information from the output.

Reply:
`-cxx` is specific to the mpich wrapper. I forgot to mention we could use `MPICH_CXX`, `OMPI_CXX` and `I_MPI_CXX` instead. I can add this if I manage to convince you to generalise the compilation flow.

Reply:
I do not think this is the right way to do it. Where are the `MPICH_CXX` etc. flags used? I only see them here.
I recommend you do something like: add a `--alternate-mpi` with default value of `none`. Then replace

    f.write("CC = mpicxx -cxx="+args.comp+"\n")

with 

    if not args.alternate_mpi: f.write("CC = mpicxx -cxx="+args.comp+"\n")
    else if args.alternate_mpi = "openmpi":  [ print a CC macro that works for openmpi]
    ....

and additional statements appropriate to each alternate MPI variant.

Reply:
I wasn't finished, hold on please. :)

Reply:
Looking a bit more like it! I just want to run some tests now ;)